awx/ui_next/src/screens/ExecutionEnvironment/ExecutionEnvironmentList/ExecutionEnvironmentList.jsx

```diff
@@ -147,7 +147,7 @@ function ExecutionEnvironmentList({ i18n }) {
                 showSelectAll
                 isAllSelected={isAllSelected}
                 onSelectAll={isSelected =>
-                  setSelected(isSelected ? [...executionEnvironments] : [])
+                  setSelected(isSelected ? [...s] : [])
```

Comment:
Were the changes to these js files intentional?

Reply:
oh wow, it was definitely not, sorry about that